src/plugins/kernel_install/utils.sh

```diff
@@ -197,7 +197,7 @@ function collect_deploy_info()
   bootloader="[bootloader]=$bootloader"
 
   # Get distro
-  distro=$(cat /etc/*-release | grep -w 'ID\(_LIKE\)\?' | cut -d = -f 2 | xargs echo)
+  distro=$(grep < /etc/*-release -w 'ID\(_LIKE\)\?' | cut -d = -f 2 | xargs echo)
```

Comment:
Since you are adjusting codestyle in this part, also address:

https://kworkflow.org/content/codingstyle.html#avoid-short-options
https://kworkflow.org/content/codingstyle.html#use-printf-instead-of-echo